src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -133,6 +133,14 @@ internal class GrpcWorkerChannel : IRpcWorkerChannel, IDisposable
             _state = RpcWorkerChannelState.Default;
         }
 
+        public Process Process
+        {
+            get
+            {
+                return (_rpcWorkerProcess as RpcWorkerProcess).Process;
```

Comment:
If this cast fails, it will result in a null reference. If this type is expected to _always_ be `RpcWorkerProcess`, use an explicit cast: `((RpcWorkerProcess)_rpcWorkerProcess).Process;`

An `InvalidCastException` will be less ambiguous than `NullReferenceException`.

Reply:
I added `IWorkerProcess` to `IWorkerChannel  - it is much cleaner now.